### PR TITLE
static auth changes for node-exporter

### PR DESCRIPTION
**Pull request by prashbnair**
* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: pkg/manifests/manifests.go [hunk 1]
```diff
@@ -774,6 +775,17 @@ func (f *Factory) NodeExporterPrometheusRule() (*monv1.PrometheusRule, error) {
 	return f.NewPrometheusRule(f.assets.MustNewAssetReader(NodeExporterPrometheusRule))
 }
 
+func (f *Factory) NodeExporterRBACProxySecret() (*v1.Secret, error) {
```
Comment (arajkumar) [inline pkg/manifests/manifests.go:778]: Should we add this to TestUnconfiguredManifests test?

/hold

Comment (prashbnair) [reply pkg/manifests/manifests.go:778]: Done

**Review by arajkumar — APPROVED** [review]
/lgtm

**Pull request merged**